SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -225,7 +225,7 @@ bool InterestFactory::DoOwnersHaveEntityId(const AActor* Actor) const
 {
 	AActor* Owner = Actor->GetOwner();
 
-	while (Owner != nullptr && !Owner->IsPendingKillPending())
```

Comment:
is it correct that the function will return true in the case that any of the conditions on the while loop causes an abort of the loop (which can mean the owner does not have an entity id)?

Reply:
If the loop aborts before reaching the top, it means we reached an owner that we would not want to consider (pending kill, or not replicated).
So we checked all the owners that mattered before returning true.